tests are failing on main and develop

Just running the tests on the main or develop branch and I am seeing this error and getting one failing test

```bash
    @given(tokamak_source=tokamak_source_strategy())
    @settings(max_examples=50)
    def test_strengths_are_normalised(tokamak_source):
        """Tests that the sum of the strengths attribute is equal to"""
>       assert pytest.approx(sum(tokamak_source.strengths)) == 1
E       assert nan ± ??? == 1
E         comparison failed
E         Obtained: 1
E         Expected: nan ± ???
E       Falsifying example: test_strengths_are_normalised(
E           tokamak_source=TokamakSource(angles=(0, 6.283185307179586), elongation=1.0, ion_density_centre=1.09e+20, ion_density_peaking_factor=1, ion_density_pedestal=1.09e+20, ion_density_separatrix=3e+19, ion_temperature_beta=6, ion_temperature_centre=45.9, ion_temperature_peaking_factor=8.06, ion_temperature_pedestal=6.09, ion_temperature_separatrix=0.1, major_radius=1.0, minor_radius=1.5258789062500003e-05, mode='H', name='TokamakSource', pedestal_radius=1.52587890625e-05, sample_size=1000, triangularity=0.0),
E       )
```

--- a/tests/test_tokamak_source.py
+++ b/tests/test_tokamak_source.py
@@ -301,7 +301,7 @@ def tokamak_source_strategy(draw):
         shafranov_factor=shafranov_factor,
         ion_density_centre=1.09e20,
         ion_density_peaking_factor=1,
-        ion_density_pedestal=1.09e20,
+        ion_density_pedestal=1.01e20,
         ion_density_separatrix=3e19,
         ion_temperature_centre=45.9,
         ion_temperature_peaking_factor=8.06,

